dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/SetBillingAddressOnCartTest.php

```diff
@@ -403,6 +403,51 @@ public function testSetBillingAddressIfCustomerIsNotOwnerOfAddress()
         $this->graphQlQuery($query, [], '', $this->getHeaderMap('customer2@search.example.com'));
```

Comment:
Need to add the same case for Guest
https://github.com/magento/graphql-ce/blob/3469f4b251cdc4874048dfff0374f818cb757964/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Guest/SetBillingAddressOnCartTest.php

Reply:
@naydav Yes, I will do that